src/transports/ws/bws.c

```diff
@@ -314,6 +344,71 @@ static void nn_bws_handler (struct nn_fsm *self, int src, int type,
         }
 
 /******************************************************************************/
+/*  CLOSING_USOCK state.                                                      */
```

Comment:
That said, I think Martin used these block comments extensively elsewhere in the code.  They still annoy me, and I celebrate a little when I can remove them. :-)